dojo/forms.py

```diff
@@ -484,7 +486,7 @@ class ScanSettingsForm(forms.ModelForm):
         help_text=addHelpTxt,
         validators=[
             validators.RegexValidator(
-                regex='^\s*([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+,*\s*)+\s*$',
+                regex=r'^\s*([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+,*\s*)+\s*$',
```

Comment:
@propersam was this change intentional?

Reply:
Yes, It was flagging flake8 error at the '\'. adding 'r' solves it.